layouts/partials/header.html

```diff
@@ -1,12 +1,10 @@
 {{ $isHomePage := eq .Title .Site.Title }}
-{{ $basePath := $.Scratch.Get "basePath" }}
-
 <header class="{{ if $isHomePage }}home-page{{ end }}">
   <div class="header-section">
-    <img alt="Multiformats Logo" id="logo" src="{{ $basePath}}logo.svg"/>
+    <img alt="Multiformats Logo" id="logo" src="{{ .Site.BaseURL }}logo.svg"/>
```

Comment:
doesn't seem to be working, see logo missing @: https://bafybeihrib6fpgbrsaevura3prs2ckprmxdin3bscl7litbwevztosgf4u.on.fleek.co/multiaddr/

Reply:
And this one is also fixed.